.azure-pipelines/packages.yml

```diff
@@ -71,6 +71,7 @@ stages:
         packagesToPack: src/Datadog.Trace/Datadog.Trace.csproj;src/Datadog.Trace.OpenTracing/Datadog.Trace.OpenTracing.csproj
         packDirectory: $(System.DefaultWorkingDirectory)/nuget-output
         configuration: $(buildConfiguration)
+        arguments: -p:IncludeSymbols=true -p:SymbolPackageFormat=snupkg
```

Comment:
We should add this in the gitlab script (`tasks/dotnettracer.py`) as well, since we want to upload signed nuget packages soon.

Or would it be easier to add this properties to the two `csproj` files instead?

Reply:
Good point, gonna move that to the csproj

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1401/commits/15d0a5c24075eac04ecc86a1db1fd18cbd4e7570